securityc/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.10.1 as builder
+
+ENV GIT_COMMIT 9ce254d3b4c86bb3d0ecf7d27556c7ea21c08894
```

Comment:
Is this Go Build not stable enough to use `master`?

Reply:
This should be an "ARG" not an "ENV"